cmd/controller/app/controller.go

```diff
@@ -334,11 +336,14 @@ func startLeaderElection(ctx context.Context, opts *options.ControllerOptions, l
 		Identity:      id + "-external-cert-manager-controller",
 		EventRecorder: recorder,
 	}
+	// Note: Hardcoded to admin cluster for now.
+	corev1cl := corev1client.NewWithCluster(leaderElectionClient.CoreV1().RESTClient(), "admin")
```

Comment:
Hardcoding leader election to happen in admin cluster for now, since there is no preference on it yet.

Reply:
Mind tossing a TODO(kcp) on that so we can find all these spots easier?